docs/Docker.md

```diff
@@ -1,5 +1,9 @@
 # Docker
 
+Currently, Docker is not being widely used and is considered deprecated. [See this note about Docker](./local-development.md#installing-with-docker) for more details.
```

Comment:
```suggestion
Currently, Docker is not being widely used and this project's Docker configuration is considered deprecated. [See this note about Docker](./local-development.md#installing-with-docker) for more details.
```

Reply:
Thank you, I think that's a good change!